### PR TITLE
Feelings improvements

### PR DESCRIPTION
**Pull request by UwUAroze**
Resolves issue #12

*mriaow~ ;3*

### CODE REVIEW

File: src/commands/implementations/feelings/hug.ts
Comment (lilysoftpaws) [inline src/commands/implementations/feelings/hug.ts:1]: implements ICommand is redundant

File: src/utils/Randomiser.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+            this.shuffled = shuffle(this.shuffled);
+            this.currentIndex = 0;
+        }
+
+        return this.shuffled[this.currentIndex++];
+        }
+
+}
+
+export function shuffle<T>(array: T[]): T[] {
```
Comment (lilyvxv) [inline src/utils/Randomiser.ts:21]: i don't think we should let chatgpt code enter the codebase

**Review by lilysoftpaws — APPROVED** [review]
lgtm mommy .・゜✭・.・

**Comment by UwUAroze** [comment]
thanks mommy ;3

**Pull request merged**